0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -176,19 +198,12 @@ implement zero-amount transfers differently than other transfers.
 ```js
 p.send({
   id: 'd86b0299-e2fa-4713-833a-96a6a75271b8',
-  account: 'https://ledger.example/accounts/connector',
+  address: 'example.ledger.connector',
```

Comment:
I think `account` is better than `address` because address is the type/format of the value, whereas account is the thing described. Which address is it? It's the address of the account used by this ledger plugin.